de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/DesktopMessages.properties

```diff
@@ -29,3 +29,4 @@ saveSettings = Setting saved.
 savingSession = Saving Session
 savingSessionWaitNotice = Saving current session, please wait...
 requestHistoryError = Unable to retrieve request history!
+waitTime = Display warning about wait times for submitting HPC apps.
```

Comment:
I think this is used as the header to the wait time dialog?  Maybe we could change this to "Warning about HPC Wait Times" or something a little more succinct.